packages/unit-testing/stubs/ui/serverWidget/Field.js

```diff
@@ -0,0 +1,242 @@
+         */
+        this.updateDisplayType = function (options) { };
+
+        /**
+         * If Rich Text Editing is enabled, you can use this property to set the height of the rich text field (in pixels). The minimum value is 100 pixels and the maximum value is 500 pixels.
+         * @name Field#richTextHeight
+         * @type {number}
+         *
+         * @since 2015.2
+         */
+        this.richTextHeight = undefined;
+        /**
+         * If Rich Text Editing is enabled, you can use this property to set the width of the rich text field (in pixels). The minimum value is 250 pixels and the maximum value is 800 pixels.
+         * @name Field#richTextWidth
+         * @type {number}
+         *
+         * @since 2015.2
+         */
+        this.richTextWidth = undefined;
+        /**
+         * The label of the field
+         * There is a 40-character limit for custom field labels.
+         * @name Field#label
+         * @type {string}
+         *
+         * @since 2015.2
+         */
+        this.label = undefined;
+        /**
+         * There is a 40-character limit for custom field labels.
+         * @name Field#padding
+         * @type {number}
+         *
+         * @since 2015.2
+         */
+        this.padding = undefined;
+        /**
+         * Get the select options for a field
+         * The internal ID and label of the options for a select field as name/value pairs is returned.
+         * The first 1,000 available options are returned (at maximum).
+         * @restriction Server SuiteScript only
+         * @governance none
+         * @param {Object} options
+         * @param {string} [options.filter] A search string to filter the select options that are returned.
+         * @param {string} [options.filteroperator]  Supported operators are contains | is | startswith. If not specified, defaults to the contains operator
+         * @return {Array<Object>}
+         *
+         * @since 2015.2
+         */
+        this.getSelectOptions = function (options) { };
+
+        /**
+         * Set help text for a field
+         * When the field label is clicked, a popup displays the help text defined using this method.
+         * @restriction Server SuiteScript only
+         * @governance none
+         * @param {Object} options
+         * @param {string} options.help The help text for the field
+         * @param {boolean} [options.showInlineForAssistant] This means that field help will appear only in a field help popup box when the field label is clicked. The default value is false, which means the field help appears in a popup when the field label is clicked and does not appear inline.
+         * @return {void}
+         * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT when help parameter is missing
+         * @throws {SuiteScriptError} SSS_INVALID_TYPE_ARG when help parameter is not string
+         * @since 2015.2
+         */
+        this.setHelpText = function (options) { };
```

Comment:
By the docs seems that this function should return itself, so maybe the return type should be {Field}. Not sure about it.

Reply:
Not sure either, could be inconsistency in the doc as well. Looks like the functions that return Field object are mostly related to the placement of the field in the UI and this one is different. I am inclined to leave it as is for now.